handler/middleware/access-logging.go

```diff
@@ -0,0 +1,44 @@
+package middleware
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/TechBowl-japan/go-stations/common"
+)
+
+// ログに出力する構造体を定義
+type AccessLogging struct {
+	Timestamp time.Time
+	Latency   int64
+	Path      string
+	OS        string
+}
+
+// アクセス日時、リクエストパス、処理時間を出力するミドルウェア
+func AccessLoggingMiddleware(h http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// OsNameを取得
+		OsName := common.GetOsName(r)
+		// contextに格納
+		ctx := common.SetOsName(r.Context(), OsName)
+		r = r.WithContext(ctx)
+
+		// handlerのアクセス時刻を取得
+		start := time.Now()
+		// ハンドラに渡す
+		h.ServeHTTP(w, r)
+		// handlerの処理時間を取得
+		duration := time.Since(start)
+		// AccessLoggingにアクセス日時、処理時間、リクエストパスを格納
+		al := AccessLogging{
+			Timestamp: start,
+			Latency:   duration.Milliseconds(),
+			Path:      r.URL.Path,
+			OS:        OsName,
+		}
+
+		fmt.Println(al)
```

Comment:
構造体を出力するときは、 `%#v` や `%+v` を利用するとフォーマットが見やすくなります！

```suggestion
		fmt.Printf("%+v\n", al)
```